eng/code-quality-reports/src/main/resources/checkstyle/checkstyle-suppressions.xml

```diff
@@ -100,6 +100,7 @@
   <suppress checks="IllegalImport" files=".*[/\\]com[/\\]azure[/\\]identity[/\\]*"/>
   <suppress checks="IllegalImport" files="com.azure.messaging.servicebus.TracingIntegrationTests.java"/>
   <suppress checks="IllegalImport" files="com.azure.messaging.eventhubs.TracingIntegrationTests.java"/>
+  <suppress checks="IllegalImport" files="com.azure.messaging.eventhubs.PublishEventsTracingWithCustomContextSample.java"/>
```

Comment:
I am curious, what is the illegal import it doesn't like?

Reply:
The sample imports OTel stuff and we disallow it in https://github.com/lmolkova/azure-sdk-for-java/blob/4a77508c86ae1f1698f86cecd65986129892062b/eng/code-quality-reports/src/main/resources/checkstyle/checkstyle.xml#L66-L67

It's a necessary check for library code, but not for samples/tests